doc/docs/api/description.md

```diff
@@ -41,6 +41,10 @@ Note that all properties also come with change events, so you can easily track w
 
     Disables the collection of metrics on all agents.
 
+  * `SetLogLevel(in s loglevel)`
+
+    Set new log level for hirte manager.
```

Comment:
Could you mention that this change is not being persisted and is only effective as long as hirte/-agent is not restarted?